LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/WebSocketWriterRegistry.cs

```diff
@@ -67,21 +67,14 @@ public WebSocketWriterRegistry(ILogger<WebSocketWriterRegistry<TKey, TMessage>>?
         /// </summary>
         public bool TryGetHandle(TKey key, [NotNullWhen(true)] out IWebSocketWriterHandle<TMessage>? handle)
         {
-            var keyFound = TryGetValue(key, out var value);
-            handle = value?.Handle;
-            return keyFound;
+            lock (this.sockets)
```

Comment:
was this a bug? If so, should this be tracked separately?

Reply:
No, it was not a bug, only a refactoring happened here. See 7f8f9bfb54deb2a49028d1c5194ea0ada4db9cc1 for the full context.

Previously we had a helper private method called `TryGetValue` called from 2 points `IsSocketWriterOpen` and `IsSocketWriterOpen`. Since we decided to drop the connectivity check in the ConcentratorDeduplication, we can delete `IsSocketWriterOpen` which means the helper method `TryGetValue` is used from only one place so can be inlined.

Reply:
Makes sense.